bpf/init.sh

```diff
@@ -611,6 +611,8 @@ if [ "$HOSTLB" = "true" ]; then
 		bpf_load_cgroups "$COPTS" bpf_sock.c bpf_sock.o sockaddr connect6 $CALLS_MAP $CGROUP_ROOT $BPFFS_ROOT
 		if [ "$HOSTLB_PEER" = "true" ]; then
 			bpf_load_cgroups "$COPTS" bpf_sock.c bpf_sock.o sockaddr getpeername6 $CALLS_MAP $CGROUP_ROOT $BPFFS_ROOT
+		else
+			bpf_clear_cgroups $CGROUP_ROOT getpeername6
```

Comment:
This part at least is not necessary. We load it unconditionally if the kernel probe was successful similar to the `connect{4,6}`, meaning there is no option where agent could be restarted in between where this else branch would ever trigger.